src/gurobi_optimods/opf/grbformulator_ac.py

```diff
@@ -905,7 +948,7 @@ def lpformulator_ac_create_constraints(alldata, model):
         lpformulator_ac_add_polarconstraints(alldata, model)
 
     # nonconvex e, f representation
-    if alldata["use_ef"]:
+    if alldata["skipjabr"] is False and alldata["use_ef"]:
```

Comment:
Why `is False`? `if not alldata["skipjabr"]` is the proper way to do a boolean test in python